README.md

```diff
@@ -39,12 +38,18 @@ In Kibana, click the "Create" button, then the "Discover" tab in the
 left-hand navigation pane.
 
 ### Redis input
-If you have the [redis-cli][redis-cli] installed, you can try injecting messages into Redis,
+You can also telnet/nc to the redis port and try injecting messages into Redis (alternatively, use [redis-cli](http://redis.io/topics/quickstart)),
 which Logstash will then ingest into Elasticsearch. Like this:
 
 ``` shell
```

Comment:
Cool. I had a telnet example in earlier but I thought it was too techy. Let's do it.